pr_agent/settings/pr_sort_code_suggestions_prompts.toml

```diff
@@ -0,0 +1,46 @@
+[pr_sort_code_suggestions_prompt]
+system="""
+"""
```

Comment:
**Suggestion:** The 'system' string in the 'pr_sort_code_suggestions_prompt' section is empty. Consider adding a brief description of the system's role or purpose in this context.
```suggestion
system="""You are a language model tasked with sorting code suggestions based on their importance."""
```